python/ray/tune/trainable/trainable.py

```diff
@@ -197,25 +197,34 @@ def __init__(
         log_sys_usage = self.config.get("log_sys_usage", False)
         self._monitor = UtilMonitor(start=log_sys_usage)
 
-        self.remote_checkpoint_dir = remote_checkpoint_dir
-        # If no sync_config is provided, but we save to a remote_checkpoint_dir,
-        # then provide a default syncer. `upload_dir` here is just a dummy directory
-        # that tells the SyncConfig to create a default syncer.
-        self.sync_config = sync_config or SyncConfig(
-            upload_dir=self.remote_checkpoint_dir, syncer="auto"
-        )
+        # TODO(justinvyu): [code_removal] These attrs can be removed.
+        if _use_storage_context():
+            self.remote_checkpoint_dir = None
+            self.sync_config = None
```

Comment:
Should these be `None` or the values that are passed in (without defaults)?

Reply:
Should be None to make sure we're not using these legacy attrs in the new codepath anymore.